tests/moduletool/test_add.py

```diff
@@ -294,7 +294,7 @@ def test_module_add_preinstalled_v1(snippetcompiler_clean, caplog) -> None:
             f"Currently installed {module_name}-2.0.0 does not match constraint ~=2.1.0: updating to compatible version."
             in caplog.messages
         )
-    assert ModuleTool().get_module(module_name).version == Version("2.1.10")
+    assert Version("2.1.11") <= ModuleTool().get_module(module_name).version < Version("2.2")
```

Comment:
I don't get this. Wouldn't this just remain as is? If we set the constraint to `<2.1.11` the version should always be `2.1.10`, shouldn't it? At the very least it should never be `>= 2.1.11` as this assertion seems to imply.

Reply:
The additional constraint is a different testcase 

Here I just extended the assertion to be match the semantic  bound

Reply:
Oh, I had misinterpreted that. Fine by me!

Reply:
I could apply that solution here as well?

Reply:
No, I think this asserts the essence of what the test tries to verify.